src/codegen/mod.rs

```diff
@@ -2278,19 +2278,28 @@ impl<'a> EnumBuilder<'a> {
                 }
             }
 
-            EnumBuilder::Bitfield { .. } => {
-                let constant_name = match mangling_prefix {
-                    Some(prefix) => {
-                        Cow::Owned(format!("{}_{}", prefix, variant_name))
-                    }
-                    None => variant_name,
-                };
-
-                let ident = ctx.rust_ident(constant_name);
-                result.push(quote! {
-                    #doc
-                    pub const #ident : #rust_ty = #rust_ty ( #expr );
-                });
+            EnumBuilder::Bitfield { canonical_name, .. } => {
+                if ctx.options().rust_features().associated_const {
```

Comment:
So we need to guard this condition to prevent entering here if the enum is unnamed.

Reply:
How can we check? I see "_bindgen_ty_{}" in `Item::base_name` but am not sure how this information is propagated further. (I'm not sure whether we care about cases where the user actually defines a type called `_bindgen_ty_1` or such.)

Reply:
Unnamed enums can be detected using `ir::Type::name`, in which case the `name would be `None`.